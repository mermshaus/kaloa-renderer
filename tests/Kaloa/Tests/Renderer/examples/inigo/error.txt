Nested in wrong way: [i]wrong closing element[/b]
