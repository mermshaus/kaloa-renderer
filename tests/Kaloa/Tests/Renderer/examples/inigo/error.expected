<p>Nested in wrong way: <em>wrong closing element</strong></p>